src/pages/Block/Tabs/OverviewTab.tsx

```diff
@@ -20,20 +22,63 @@ function VersionValue({data}: {data: Types.Block}) {
   );
 }
 
+function BlockMetadataRows({
+  blockTxn,
+}: {
+  blockTxn: Types.Transaction | undefined;
+}) {
+  if (!blockTxn) {
+    return null;
+  }
+
+  const txn = blockTxn as Types.Transaction_BlockMetadataTransaction;
+
+  return (
+    <>
+      <ContentRow
+        title="Proposer:"
+        value={<HashButton hash={txn.proposer} type={HashType.ACCOUNT} />}
+        tooltip={getLearnMoreTooltip("proposer")}
+      />
+      <ContentRow
+        title="Round:"
+        value={txn.round}
+        tooltip={getLearnMoreTooltip("round")}
+      />
+    </>
+  );
+}
+
 type OverviewTabProps = {
   data: Types.Block;
 };
 
 export default function OverviewTab({data}: OverviewTabProps) {
+  const blockTxn: Types.Transaction | undefined = (
+    data.transactions ?? []
+  ).find((txn) => {
```

Comment:
(For some reason, this comment didn't get posted)

Instead of casting it to the type in the component, it would be better to create a type guard (i.e. `is` instead of `as`).

https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

Reply:
Thanks for the tip! Applied changes in #174